submissions/Temu4/tiny-js-world/index.js

```diff
@@ -0,0 +1,62 @@
+// ======== OBJECTS DEFINITIONS ========
+const man = {
+  species: 'human',
+  gender: 'male',
+  name: 'John',
+  age: 28,
+  hands: 2,
+  legs: 2,
+  hobby: 'fishing',
+  speach: 'Hello, people!'
+};
+
+const woman = {
+  species: 'human',
+  gender: 'female',
+  name: 'Anny',
+  age: 23,
+  hands: 2,
+  legs: 2,
+  hobby: 'cycling',
+  speach: 'Hello, everybody!'
+};
+
+const dog = {
+  species: 'dog',
+  gender: 'male',
+  name: 'Rex',
+  age: 3,
+  hands: 0,
+  legs: 4,
+  hobby: 'diging bones',
+  speach: 'Wohw!'
+};
+
+const cat = {
+  species: 'cat',
+  gender: 'female',
+  name: 'Sida',
+  age: 1.5,
+  hands: 0,
+  legs: 4,
+  hobby: 'sleaping',
+  speach: 'Meow!'
+};
+
+//Create message from object properties
+const createMessage = obj => {
+  let msg1 = `\t${obj.speach} This is <strong>${obj.name}</strong>. `;
+  let msg2 = ` is a ${obj.species} (with ${obj.hands} hands and ${
+    obj.legs
```

Comment:
Also remember that we write code for human, for pretty read. At first sight this part with new lines are redudant.

Reply:
It's only config of my IDE of line size (80 symbols, that is suitable for my screen).

Reply:
fix in [this](https://github.com/kottans/frontend-2019-p2p/pull/44/commits/38702b7912088fa4d925c81da09dada474a2adbf) commit.